scaaml/capture/aes_capture.py

```diff
@@ -0,0 +1,206 @@
+from pathlib import Path
+
+from scaaml.capture.aes_capture_util import *
+from scaaml.capture.acqktp import AcqKeyTextPattern_Scaaml as ktp_scaaml
+from scaaml.io import Dataset
+from scaaml.io.resume_kti import ResumeKTI
+
+
+class AESCaptureRunner:
+    def __init__(
+        self,
+        firmware_hash:
+        str = '18ba5773bbd766d5e6f76a6f1ebb689570ebed679232e22c8cf9ddbd140196d1',
```

Comment:
I don't see the point about having a hardcoded sha256.
A few other default value don't really make sense here neither.

Reply:
Good point. I made it into a parameter.

As for other default values. I am happy to change those. Can you please elaborate which do not make sense and why?